### PR TITLE
Complete lint for pkg/slices

### PR DESCRIPTION
**Pull request by Wind0r**
Complete lint for pkg/slices.

- [ ] CHANGELOG.md updated
- [ ] Rebased/mergable
- [ ] Tests pass
- [ ] Sign [CLA](http://influxdb.com/community/cla.html) (if not already signed)

### CODE REVIEW

File: pkg/slices/strings.go [hunk 1]
```diff
@@ -2,21 +2,22 @@ package slices
 
 import "strings"
 
+// Union combines to string sets
```
Comment (corylanou) [inline pkg/slices/strings.go]: nit: `to` -> `two`

File: pkg/slices/strings.go [hunk 2]
```diff
@@ -2,21 +2,22 @@ package slices
...
 func Union(setA, setB []string, ignoreCase bool) []string {
+	exists := Exists
```
Comment (corylanou) [inline pkg/slices/strings.go]: This feels odd.  Can you explain your rational for this?

Comment (corylanou) [inline pkg/slices/strings.go]: Ok, I think I see why, but I think I prefer the way it was.  If you wanted to refactor Exists to be a single method with `ignoreCase` as a parameter that might be a better way to handle this.

**Comment by corylanou** [comment]
Nit and some questions.

**Comment by otoolep** [comment]
@Wind0r -- have you signed the CLA?

**Comment by otoolep** [comment]
CLA confirmed signed, and these changes are now only comments -- merging now.

Thanks @Wind0r

**Pull request merged**